### PR TITLE
Update UG with quiz feature and update index.md

### PR DESCRIPTION
**Pull request by moyj01**

### CODE REVIEW

**Review by tan-zhuoyao — CHANGES_REQUESTED** [review]
Would be good if you can add your quiz feature into the command summary 👍

File: docs/UserGuide.md [hunk 1]
```diff
@@ -209,6 +212,24 @@ Examples:
...
+
+Quiz the current list of flashcards. This puts the user in quiz mode and the user can no longer input commands to the textbox.
+
+Upon entering quiz mode, the following user input will be recognised:
+* `↓ key` shows answer and notes of the current flashcard  
+* `q` key quits review mode
+* `y` to represent correct answer. This input will only be recognised after the `↓ key` is pressed
+* `n` to represent incorrect answer. This input will only be recognised after the `↓ key` is pressed
+
+Upon pressing the down arrow key, the user will be prompted if they got the answer correct. The user can then press
```
Comment (sgn00) [inline docs/UserGuide.md]: would be good to use ↓ instead of down arrow key

File: docs/UserGuide.md [hunk 2]
```diff
@@ -209,6 +212,24 @@ Examples:
...
     - `q` quits review mode
+
+### Quizzing flashcards: `quiz`
+
+Quiz the current list of flashcards. This puts the user in quiz mode and the user can no longer input commands to the textbox.
+
+Upon entering quiz mode, the following user input will be recognised:
+* `↓ key` shows answer and notes of the current flashcard  
+* `q` key quits review mode
+* `y` to represent correct answer. This input will only be recognised after the `↓ key` is pressed
```
Comment (sgn00) [inline docs/UserGuide.md]: maybe should make it more clear that y/n is feedback not giving the answer

**Review by sgn00 — CHANGES_REQUESTED** [review]
Refer to comments

**Pull request merged**